test/scripts/selector-declaration-builder.test.js

```diff
@@ -0,0 +1,101 @@
+const $wDeclarationBuilder = require("../../scripts/selector-declaration-builder");
+
+describe("define $w.d.ts", () => {
+  function get$wDts({
+    servicePath,
+    tripleSlashDirectivePath = "directive"
+  } = {}) {
+    const services = [require("./services/$w.service.json")];
+    if (servicePath) {
+      services.push(require(servicePath));
+    }
+
+    return $wDeclarationBuilder(services, tripleSlashDirectivePath);
+  }
+
+  describe("triple slash directive", () => {
+    test("should generate triple slash directive with the given path", () => {
+      const tripleSlashDirectivePath = "../common/utilityTypes.d.ts";
+      const dts = get$wDts({ tripleSlashDirectivePath });
+      const expectedDeceleration = ``; /// <reference path="${tripleSlashDirectivePath}" />`
```

Comment:
This is currently an empty string, I am assuming not on purpose

Reply:
Oops, I removed this flow